extensions/amp-story/1.0/amp-story.js

```diff
@@ -2625,6 +2629,22 @@ export class AmpStory extends AMP.BaseElement {
     );
   }
 
+  /**
+   * Handles the rewind viewer event.
+   * @private
+   */
+  onRewind_() {
+    this.signals()
+      .whenSignal(CommonSignals.LOAD_END)
+      .then(() => {
+        if (this.pages_.length > 0) {
+          return;
+        }
+        return this.initializePages_();
+      })
```

Comment:
from reading the code, I don't see a code path where pages wouldn't be initialized after `LOAD_END` signal. Is this needed?

Reply:
Yep, I think that piece was outdated. Thanks for pointing it out.